README.md

```diff
@@ -111,12 +130,34 @@ $subscriber->setSubscribed();
 $subscriber->setPending();
 ```
 
-## YMLP options
+### Mailchimp options sender transactional email
+
+See https://mailchimp.com/developer/transactional/api/messages/send-using-message-template/
+
+```php
+$subscriber = $factory->createFor('mailchimp');
+// Mailchimp only requires an API Key
+$subscriber->setApiKey('mailchimp_api_key');
+// use an array of value object EmailAddress for recipients
+$emails[] = new \SubscribeMe\ValueObject\EmailAddress('hello@super.test', 'John Doe');
+// Mailchimp only use string for his $templateEmailId
+$emailTemplateId = 'teamplate_name';
+// Mailchimp use an array of variables arrays how contains 'name' and 'content'
```

Comment:
Are your sure of that ?
Your MailchimpSubscriber do not transform the variables array into the form wanted by Mailchimp?

The purpose of this library is to make transactional email sending generic. So we should require   simple *key-value* array as inputs. Then each subscribers transform it.